### PR TITLE
Use the recommended apt format to install the virtualenv pkgs

### PR DESCRIPTION
**Pull request by danypr92**

### CODE REVIEW

File: roles/virtualenv/tasks/main.yml [hunk 1]
```diff
@@ -2,13 +2,8 @@
...
   become: yes
   apt:
-    pkg={{ item }}
-    state=present
-  with_items:
-    - python-dev=2.7.13-2
-    - python-pip=9.0.1-2
-    - python-virtualenv=15.1.0+ds-1
-    - python-setuptools=33.1.1-1
+    pkg: ['python-dev=2.7.13-2', 'python-pip=9.0.1-2', 'python-virtualenv=15.1.0+ds-1', 'python-setuptools=33.1.1-1']
```
Comment (enricostano) [inline roles/virtualenv/tasks/main.yml]: Can we still have multiline? will be easier do code review in the future.

Comment (danypr92) [reply roles/virtualenv/tasks/main.yml]: Applied!

Comment (enricostano) [reply roles/virtualenv/tasks/main.yml]: I still see it all in one line. Cache?

Comment (danypr92) [reply roles/virtualenv/tasks/main.yml]: image

I see the pkgs in multiline :sweat_smile:

Comment (enricostano) [reply roles/virtualenv/tasks/main.yml]: I'm talking about `roles/virtualenv/tasks/main.yml`, you're looking at `roles/system_packages/tasks/main.yml`

Comment (danypr92) [reply roles/virtualenv/tasks/main.yml]: Sorry, I was wrong. Now we format the Python packages in multiline. :sweat_smile:

**Pull request merged**